website/src/_posts/2019-04-liftoff-09.md

```diff
@@ -7,6 +7,8 @@ series: 30 Days to Liftoff
 seriesSuffix: 'of 30'
 ---
 
+<!--lint disable no-undefined-references-->
+
```

Comment:
Is this necessary? All blog posts before July 2021 are ignored by the linter (well at least I thought that is what I set up)

Reply:
I couldn't push without adding this 🤷‍♂️